Week06/StudentDesc.md

```diff
@@ -11,7 +11,7 @@ Learn how to use CSS to add style to webpages. Review the two PowerPoint present
 <iframe src='https://view.officeapps.live.com/op/embed.aspx?src=https://hylandtechclub.com/web-101/Week06/CssSelectorsAndProperties.pptx' width='100%' height='450px' frameborder='0'></iframe>
 
 ## Code-Along
-[Click here for the code-along **starter code**.](https://replit.com/@HylandOutreach/BananaStarter#index.html)
+[Click here for the code-along **starter code**.](https://glitch.com/edit/#!/bananastarter)
```

Comment:
for example, this can also use the `/remix` link - just add `/remix` right after the `#!`